src/chatd.cpp

```diff
@@ -4279,6 +4297,7 @@ void Chat::handleTruncate(const Message& msg, Idx idx)
     CHATID_LOG_DEBUG("Truncating chat history before msgid %s, idx %d, fwdStart %d", ID_CSTR(msg.id()), idx, mForwardStart);
     CALL_CRYPTO(resetSendKey);      // discard current key, if any
     CALL_DB(truncateHistory, msg);
+    mOldestIdxInDb = mDbInterface->getOldestIdx();
```

Comment:
Isn't now the oldest index in DB the `idx`?? We've just truncated all previous history. Why should we do a DB query then?